plugins/catalog-backend/src/database/types.ts

```diff
@@ -52,6 +52,12 @@ export type DbLocationsRow = {
   target: string;
 };
 
+export type DbLocationsRowWithStatus = DbLocationsRow & {
+  status: DatabaseLocationUpdateLogEvent['status'] | null;
+  timestamp: DatabaseLocationUpdateLogEvent['created_at'] | null;
+  message: DatabaseLocationUpdateLogEvent['message'] | null;
```

Comment:
it's probably clearer to just write out the types

Reply:
- [x] write out the types explicitly